src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsAdminList.cshtml

```diff
@@ -1,6 +1,6 @@
 @model ListContentsViewModel
 
-<script asp-name="bootstrap-select" depends-on="bootstrap" at="Foot"></script>
```

Comment:
This is great now Bootstrap.js is compiled as part of TheAdmin.js so we don't need this anymore else it will load it twice in the page which will create issues.

Reply:
This technically means also that we could start using something else than Bootstrap for some pages. It won't conflict as these are now scoped.